privacyidea/lib/tokenclass.py

```diff
@@ -1289,19 +1287,6 @@ def get_init_detail(self, params=None, user=None):
         response_detail.update(init_details)
         response_detail['serial'] = self.get_serial()
 
-        otpkey = None
```

Comment:
Please do not mix up topics in pull requests.
Besides - I think this is correct here, since other tokens could need a secret key, too.

Reply:
done